### DISCUSSION TITLE
Airflow 2.7.0 - sync_users.py fails

### DISCUSSION THREAD

**Discussion by maciejmochol** [airflow-helm/charts#771] [Questions & Answers]
Hello! First let me thank you all for this excellent Airflow helm chart.

Airflow 2.7.0 is out and i can see that airflow-sync-users pod fails now with error. It looks that file models.py moved from airflow.www.fab_security.sqla folder... What can I do to fix it?

`
_Traceback (most recent call last):
  File "/mnt/scripts/sync_users.py", line 146, in <module>
    from airflow.www.fab_security.sqla.models import User, Role
ModuleNotFoundError: No module named 'airflow.www.fab_security.sqla.models'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/scripts/sync_users.py", line 148, in <module>
    from flask_appbuilder.security.sqla.models import User, Role
  File "/home/airflow/.local/lib/python3.11/site-packages/flask_appbuilder/security/sqla/models.py", line 48, in <module>
    assoc_permissionview_role = Table(
                                ^^^^^^
  File "<string>", line 2, in __new__
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py", line 375, in warned
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/sqlalchemy/sql/schema.py", line 596, in __new__
    raise exc.InvalidRequestError(
sqlalchemy.exc.InvalidRequestError: Table 'ab_permission_view_role' is already defined for this MetaData instance.  Specify 'extend_existing=True' to redefine options and columns on an existing Table object._
`

**Comment by thesuperzapper** [comment]
@maciejmochol thanks for raising this issue, it will be fixed by PR #772, and it will be in the next release of the chart! Please discuss any further concerns in that thread.

That is to say, chart version 8.7.1 does NOT support Airflow 2.7, but the next one will, please see the [Airflow Version Support Matrix](https://github.com/airflow-helm/charts/tree/main/charts/airflow#airflow-version-support).

**Reply by maciejmochol** [reply]
@thesuperzapper excellent, many thanks. So far we have been able to run 2.7 with current version of the chart without many problems (except the issue with sync_users). The only other problem we have seen was the scheduler having some strange problem with Python imports - but restarting pod actually helped. We will of course redeploy once the new version of the chart is out.

Best regards
Maciej
